libwallet/assets/btc/dex-wallet.go

```diff
@@ -225,17 +225,27 @@ func (dw *DEXWallet) Birthday() time.Time {
 }
 
 func (dw *DEXWallet) Peers() ([]*asset.WalletPeer, error) {
-	peers := dw.spvService.CS.Peers()
-	var walletPeers []*asset.WalletPeer
-	for i := range peers {
-		p := peers[i]
-		walletPeers = append(walletPeers, &asset.WalletPeer{
-			Addr:      p.Addr(),
-			Connected: p.Connected(),
-			Source:    asset.WalletDefault,
-		})
+	peerChan := make(chan []*asset.WalletPeer)
+	go func() {
+		peers := dw.spvService.CS.Peers()
+		var walletPeers []*asset.WalletPeer
+		for i := range peers {
+			p := peers[i]
+			walletPeers = append(walletPeers, &asset.WalletPeer{
+				Addr:      p.Addr(),
+				Connected: p.Connected(),
+				Source:    asset.WalletDefault,
+			})
+		}
+		peerChan <- walletPeers
+	}()
```

Comment:
In harmony with my last comment, the required fix is for a different method but this would have also needed the fix. However, with #419, this method is no longer implemented. So once the required changes are applied to the other method, this should be good to merge.